charts/agent-control/values.yaml

```
@@ -58,6 +58,10 @@ agentControlDeployment:
   # @default -- `.Chart.appVersion`
   chartVersion:
 
+  # -- The name of the release for the CD chart.
```

```suggestion
  # -- The name of the release
```

I need to change that